api_tests/institutions/views/test_institution_auth.py

```diff
@@ -48,6 +51,14 @@ class TestInstitutionAuth:
     def institution(self):
         return InstitutionFactory()
 
+    @pytest.yield_fixture(autouse=True)
+    def flask_request_context(self):
+        """
+        required for waffle cookies
+        """
+        with decoratorapp.test_request_context():
+            yield
```

Comment:
These tests run the `authenticate` method of `InstitutionAuthentication` which checks the Flask request object [here](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/institutions/authentication.py#L115). `test_request_context` ensure that the Flask request that `flag_is_active` needs will be there, otherwise we get a conflict.